src/Notification/ToastNotification.svelte

```diff
@@ -55,6 +55,11 @@
   let timeoutId = undefined;
 
   function close(closeFromTimeout) {
+    if (closeFromTimeout !== true && timeoutId) {
+      // If the close button is clicked, clear the timeout.
+      clearTimeout(timeoutId);
+    }
```

Comment:
The `if` wrapper seems a bit redundant. `clearTimeout` does nothing on expired timeout handler or `undefined`.